website/docs/d/cos_bucket_object.html.markdown

```diff
@@ -29,11 +29,28 @@ data "ibm_cos_bucket" "cos_bucket" {
   bucket_type          = "region_location"
   bucket_region        = "us-east"
 }
-
+data "ibm_cos_bucket" "cos_bucket_objectlock" {
+  resource_instance_id = data.ibm_resource_instance.cos_instance.id
+  bucket_name          = "my-bucket"
+  bucket_type          = "region_location"
+  bucket_region        = "us-east"
+  object_versioning {
```

Comment:
Same as above datasoure they should be computed

Reply:
addressed this comment.